utils/install-dependencies.sh

```diff
@@ -24,10 +24,9 @@ function install_dependencies_with_yum() {
     # add OpenResty source
     sudo yum install yum-utils
     sudo yum-config-manager --add-repo "https://openresty.org/package/${1}/openresty.repo"
-    sudo yum check-update
```

Comment:
Why remove this line?

Reply:
When an update is available, `yum check-update` returns status code `100`, Which will break the execution.